### ISSUE TITLE
Support import for aws_lambda_alias

### ISSUE THREAD

**Issue by pintbrewer** [terraform-providers/terraform-provider-aws#8506]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I would like the ability to import an existing lambda alias to terraform state.
My current aliases created from another IaC tool are orphaned from terraform management, but I'd like to bring them in so that they can be referenced in lambda event triggers and such.

### New or Affected Resource(s)

aws_lambda_alias

### Potential Terraform Configuration

```
terraform import aws_lambda_alias.alias1 lambda_name/alias1
```

**Comment by richardjennings** [comment]
I have opened a PR https://github.com/terraform-providers/terraform-provider-aws/pull/8513

**Comment by bflad** [comment]
Support for importing `aws_lambda_alias` resources has been merged and will release with version 2.10.0 of the Terraform AWS Provider, likely later today. Thanks, @richardjennings!

**Comment by bflad** [comment]
This has been released in [version 2.10.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2100-may-10-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
